src/test/java/org/opentripplanner/netex/mapping/StopAndStationMapperTest.java

```diff
@@ -279,6 +283,33 @@ void testDuplicateStopIndices() {
     );
   }
 
+  @Test
+  void testRouteToCentroid() {
+    var routeToCentroidStationIds = Set.of("NSR:StopPlace:1");
+    StopAndStationMapper stopMapper = new StopAndStationMapper(
+      MappingSupport.ID_FACTORY,
+      new HierarchicalVersionMapById<>(),
+      null,
+      StopModel.of(),
+      DEFAULT_TIME_ZONE,
+      DataImportIssueStore.NOOP,
+      false,
+      routeToCentroidStationIds
+    );
+
+    stopMapper.mapParentAndChildStops(
+      List.of(createStopPlace("NSR:StopPlace:1", "A", "1", 59.1, 10.0, TRAM))
+    );
+    stopMapper.mapParentAndChildStops(
+      List.of(createStopPlace("NSR:StopPlace:2", "B", "1", 59.2, 10.0, TRAM))
+    );
+
+    var stations = stopMapper.resultStations;
+    assertEquals(2, stations.size());
```

Comment:
Use truth.

Reply:
I'm not really sure how to simplify these assertions using truth. How would you do it?

Reply:
```
assertThat(stations).hasSize(2)
```

Reply:
This will give you a much nicer error message